admin_interface/static/admin_interface/css/tabbed-changeform.css

```diff
@@ -0,0 +1,28 @@
+.admin-interface .tab {
+    display: flex;
+    flex-direction: row;
+    gap: 0.75em;
+    flex-wrap: wrap;
+}
+
+.admin-interface .tab button {
+    border: none;
+    cursor: pointer;
+    padding: 8px 12px;
+    transition: 0.3s;
+    border-radius: var(--admin-interface-module-border-radius);
+}
+
+.admin-interface .tab button.active {
+    background-color: var(--admin-interface-module-background-color);
+    color: var(--admin-interface-module-text-color);
+}
+
+.admin-interface .tabcontent {
+    display: none;
+    padding: 1em 0;
+}
+
+.admin-interface .tabcontent.active {
+    display: block;
+}
```

Comment:
Sorry for the feedback in two parts on this, I think `.tab` and `.tabcontent` class names are too generic, I would prefix these classes with `.tabbed-changeform-` prefix:
- `.tabbed-changeform-tab`
- `.tabbed-changeform-tabcontent`

Better no room for doubts.

Reply:
updated